### PR TITLE
Update docs for v1.9.0

### PR DESCRIPTION
**Pull request by zhongyi-zhang**

### CODE REVIEW

**Comment by zhongyi-zhang** [comment]
@bingosummer could you help to review?

File: docs-content/release-notes.html.md.erb [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 owner: Partners
 ---
+
+### v1.9.0
+
+**Release Date:** August 20, 2018
+
+* Service broker `azure-rediscache`
+
+  * Support update API. You can update service plan to change Redis Cache instance tier. Updating the configuration `enableNonSslPort` is also supported now. See guidance [here](https://github.com/Azure/meta-azure-service-broker/blob/master/docs/azure-redis-cache.md#updating). #183
```
Comment (bingosummer) [inline docs-content/release-notes.html.md.erb]: Remove `#183`

**Review by bingosummer — CHANGES_REQUESTED** [review]

File: docs-content/release-notes.html.md.erb (hunk 1 above)
Comment (zhongyi-zhang) [reply docs-content/release-notes.html.md.erb]: Fixed. Thanks for the catch.

**Comment by jknostman3** [comment]
@zhongyi-zhang Thanks!

**Pull request merged**